DemocracyLab/CivicTechExchange
# Theme button consistency fix

On certain browser/device combinations (chrome/MacOS but not chrome/Win or chrome/Linux) some of our button elements do not render the correct, themed button, instead rendering what seems like a default 'grey' button which looks disabled and we've had user issues with it.

This fix may require a few deployments for testing, but the root cause seems to be an interaction between react-bootstrap <Button>, the Bootstrap v4 styles, and our theme button override not actually overriding properly. This first step is making sure our theme button styles override correctly when building the site stylesheet, as they were not even showing up as overridden styles when generated by sass previously, suggesting the rules were being overwritten completely.

### To test:

go to: https://democracy-lab-dev.herokuapp.com

test cases:
1) go to any project page while signed _out_ and see if the “Sign in to Contact Project” and “Sign in to Volunteer” buttons are correctly themed (orange with white text)
2) go to any project page while signed _in_ and make sure the Contact Project and Volunteer with Project buttons are still correctly themed
3) Go to any project you own and make sure the “Edit Project” button is correctly themed.

Please make sure to report OS and browser, as we want to make sure it works across all of them and doesn't break where it used to work. Also note that democracy-lab-dev may not have your saved login info; if you can only verify case 1 and not 2-3 that's fine.
